pymc/backends/arviz.py

```diff
@@ -42,6 +42,26 @@
 Var = Any  # pylint: disable=invalid-name
 
 
+def find_observations(model: Optional["Model"]) -> Optional[Dict[str, Var]]:
```

Comment:
Shouldn't that type hint be something like:
```suggestion
def find_observations(model: Optional[Model]) -> Optional[Dict[str, Var]]:
```

Where `Model` is found in `pymc.model`

Reply:
Optional["Model"] is the convention used in the rest of the file.